jaseci_core/jaseci/utils/jaseci-redis.yaml

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
@marsninja This yaml file is read into jsorc to start the jaseci pod.

Reply:
To start the redis pod actually.. :->

Reply:
👍🏼